data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+mkdir -p "${ARTIFACTS}"
```

Comment:
You'll want an earlier `set -e` or some `&&`-chaining or some such to error the script out when commands like this one fail for whatever reason.

Reply:
Also, no need for the `mkdir` here, since that's covered by the `mkdir -p "${ARTIFACTS}/bootstrap/journals"` a few lines down.